### PR TITLE
chore: service file

### PR DESCRIPTION
**Pull request by Decodetalkers**
remove dde-clipboard-daemon.service to dbus service adjust Clipboard1 service

Log: service file adjust

### CODE REVIEW

File: misc/org.deepin.dde.daemon.Clipboard1.service.in [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[D-BUS Service]
+Name=org.deepin.dde.ClipboardLoader1
```
Comment (wineee) [inline misc/org.deepin.dde.daemon.Clipboard1.service.in:2]: Why this file is same with `misc/org.deepin.dde.ClipboardLoader1.service.in`?

**Review by wineee — CHANGES_REQUESTED** [review]

File: misc/org.deepin.dde.daemon.Clipboard1.service.in (hunk 1 above)
Comment (Decodetalkers) [reply misc/org.deepin.dde.daemon.Clipboard1.service.in:2]: 搞错了

**Review by justforlxz — APPROVED** [review]

**Pull request merged**